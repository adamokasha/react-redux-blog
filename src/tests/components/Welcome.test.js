import React from 'react';
import { shallow } from 'enzyme';

import Welcome from '../../components/Welcome';

test('should render welcome component', () => {
  const wrapper = shallow(<Welcome />);
  expect(wrapper).toMatchSnapshot();
});