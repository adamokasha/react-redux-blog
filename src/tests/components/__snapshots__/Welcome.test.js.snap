// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render welcome component 1`] = `
<div>
  <WelcomeMessage />
</div>
`;
